README.md

```diff
@@ -185,12 +185,12 @@ Any deposit will be logged as a `Report` by the DApp.
 For example, a deposit performed by account `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266` on localhost using `SimplerERC20` would be similar to this:
 
 ```shell
-cast send 0x4340ac4FcdFC5eF8d34930C96BBac2Af1301DF40 \
+cast send 0xdA8A1833DC224BA41aa81Dc987fb18AeaE1256A6 \
     "depositERC20Tokens(address,address,uint256,bytes)" \
-        0xc6e7DF5E7b4f2A278906862b61205850344D4e7d \
-        0x142105FC8dA71191b3a13C738Ba0cF4BC33325e2 \
+        0x4ed7c70F96B99c776995fB64377f0d4aB3B0e1C1 \
+        0x1620a6E44Ff61CD35D91f320565853af0aD7Dfeb \
         100000000000000000000 \
-        0x00 \
+        "" \
     --rpc-url http://localhost:8545 \
     --from 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
 ```
```

Comment:
`curl localhost:5005/inspect/0` does not work to get the balance, it worked when I dropped the `0`.